### ISSUE TITLE
Link to MAPe and granular problems 404s

### ISSUE THREAD

**Issue by DaveParr** [Nixtla/statsforecast#476]
This section of hte docs: https://nixtla.github.io/statsforecast/examples/getting_started_complete.html#install-libraries:~:text=Warning,assess%20forecasting%20quality. directs to a 404. 

From url line that resolves to: `https://nixtla.github.io/statsforecast/examples/%22https://blog.blueyonder.com/mean-absolute-percentage-error-mape-has-served-its-duty-and-should-now-retire/%22` I'm guessing that this callout:

```
":::{.callout-warning}\n",
    "You can also use Mean Average Percentage Error (MAPE), however for granular forecasts, MAPE values are extremely [hard to judge](\"https://blog.blueyonder.com/mean-absolute-percentage-error-mape-has-served-its-duty-and-should-now-retire/\") and not useful to assess forecasting quality.\n",
    ":::"
```
- https://github.com/Nixtla/statsforecast/blob/main/nbs/examples/Getting_Started_complete.ipynb

needs to be reformed somehow to not prepend the Nixtla domain.

Tiny thing, but thought it might be undetected :) 

Very much enjoying playing with the package examples this morning. Going to be building something with it this afternoon :)

**Comment by FedericoGarza** [comment]
hey @DaveParr! Nice catch! #477 should fix it. Glad to hear that the library is of help. Feel free to let us know if you have any questions. :)

**Comment by DaveParr** [comment]
![dependencies](https://user-images.githubusercontent.com/8363743/236184686-dba9d26f-b5dc-4258-b37b-4ea4361e41a6.png)
Did a mega simple analysis of the package, and by eyeball half the dependencies come from `jupyter-dash`, via `plotly-resampler`. Therefore if a version of the package can have those removed for a production api environment there should be a big simplification of environments and spin up time. 

(You probably already know this though, but just in case, seeing as I had built the diagram already :) )

**Comment by FedericoGarza** [comment]
hey @DaveParr! Thanks for your questions and recommending our libraries. Your words mean a lot to us. :)

Regarding the dask integration, in the latest release, you don't have to import the `FugueBackend`, the `forecast` method can receive natively a dask dataframe. Regarding CPU/GPU usage, the methods are optimized to work efficiently only on CPU. Our [NeuralForecast](https://github.com/Nixtla/neuralforecast) library can train deep learning models for time series forecasting using GPU.

To handle errors during training, the `StatsForecast` class can receive a `fallback_model` parameter if some model fails for a specific time series (for example `StatsForecast(..., fallback_model=Naive())` will use the `Naive` model if some model fails). This parameter's limitation is that we currently don't support returning the root cause of the error or the series and models that failed.  Please don't hesitate to open an issue requesting this feature. We will be happy to work on it. 

That might be a very helpful feature! Could you help us open an issue to work on that? I think for that use case it might be really helpful to use `pip install "statsforecast[core]"`. Thank you for analyzing the dependencies of the package, we definitely can simplify it. 

I'm closing this issue related to the link (fixed in #477). Don't hesitate to open individual issues for the features you would like for us to implement. :)
